pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -165,7 +165,7 @@ func (c *controller) Get(typ, name, namespace string) (*model.Config, bool) {
 	}
 
 	ingress := obj.(*v1beta1.Ingress)
-	if !shouldProcessIngress(c.mesh, ingress) {
+	if ingress == nil || !shouldProcessIngress(c.mesh, ingress) {
```

Comment:
If certain ingress can not be found, the GetByKey func above would return error or return `exists` with false,
so I think this won't happen.

Reply:
yeah.that will not happen